pages/profiles/snigdharao.md

```diff
@@ -0,0 +1,27 @@
+ # **Snigdha Rao Madhavaram** 
+<p>
+ <img height="250" width="250" src="https://s3.amazonaws.com/mylifecylebuckhyd12/IMG_2569+(1).jpg">
+</p>
+
+Time zone : EDT, -0400
+OS : Mac osX
+BuzzWords :Python, Java, C/C++, JS, Bash, MIPS, Matlab, Linux, Postgres, MySQL, AWS
```

Comment:
Would it be better to add a space after `BuzzWords :` to make it consistent with previous 2 lines?